### PR TITLE
Polish README

### PR DESCRIPTION
**Pull request by rwinch**
General fixes to README

* Change links to asciidoc format
* Update links to use new repository location
* Update travis build status

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Do not edit this file (e.g. go instead to src/main/asciidoc)
```
Comment (spencergibb) [inline README.adoc]: So, you want to make these changes in `src/main/asciidoc/README.adoc`

Comment (rwinch) [inline README.adoc]: Helps if I read the comments. Where does the Lombok change go then? I only see it in the root README.adoc

Comment (spencergibb) [inline README.adoc]: [Here](https://github.com/spring-cloud/spring-cloud-build/blob/master/docs/src/main/asciidoc/building-base.adoc), I'll get that one.

Comment (spencergibb) [inline README.adoc]: Done https://github.com/spring-cloud/spring-cloud-build/commit/cbc9c0232efd8f384616082886ecee13d1be63e0

Comment (rwinch) [inline README.adoc]: Thanks! I pushed the changes to docs/src/main/asciidoc/README.adoc

**Pull request merged**